packages/gui/src/components/html/Provider.tsx

```diff
@@ -28,6 +31,30 @@ export function useHtmlEditor() {
 
 const HtmlEditorContext = createContext<HtmlEditor>(DEFAULT_HTML_EDITOR_VALUE)
 
+const transformFunction = (property: string, value: any) => {
```

Comment:
This seems like it could live inside `stylesToEditorSchema` rather than being a function passed as a parameter?